packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -74,13 +84,24 @@ const ShowHide = (props) => {
       intlButtonText = intl.formatMessage({ id: 'Terra.showhide.showmore' });
     }
   }
+  /**
+   * This prevents container from getting focus on click.
+   * Does not prevent container's children to receive focus and be click-able.
+   */
 
   return (
     <div {...customProps}>
       {!isOpen && preview}
-      <Toggle isOpen={isOpen}>
-        {children}
-      </Toggle>
+      <div
+        className={cx(['show-hide', 'content-container', theme.className])}
+        tabIndex={-1}
```

Comment:
nit: In other cases in the code, I'm seeing this as a numeric string when assigning the tabIndex

Reply:
Good catch, fixed in [this commit](https://github.com/cerner/terra-core/pull/3830/commits/9e8aff2a397aa02be6fda129d3f049612e78d1e0)